charts/opentelemetry-demo/values.yaml

```diff
@@ -48,6 +48,9 @@ default:
     affinity: {}
     tolerations: []
   securityContext: {}
+  # list of pod labels applied to all components
+  podLabels: {}
+  # sidecar.istio.io/inject: true
```

Comment:
If you need it for istio, I believe the best practice is using annotations, not labels. I didn't even know that labels supported the same way

Reply:
I hope I can use annotations, but I found this way is [Deprecated](https://istio.io/latest/docs/reference/config/annotations/). [Controlling the injection policy](https://istio.io/latest/docs/setup/additional-setup/sidecar-injection/#controlling-the-injection-policy) tell me to use lables.